testing/src/context.py

```diff
@@ -41,14 +41,8 @@
     _Event,
 )
 
-# TODO: Remove the line below after the ops compatibility code is removed.
-# pyright: reportUnnecessaryTypeIgnoreComment=false
-
 if TYPE_CHECKING:  # pragma: no cover
-    try:
-        from ops._private.harness import ExecArgs  # type: ignore
-    except ImportError:
-        from ops.testing import ExecArgs  # type: ignore
+    from ops._private.harness import ExecArgs
 
     from scenario.ops_main_mock import Ops
```

Comment:
Do we want this blank line?

Reply:
This one should be here, because we consider `ops` to be third-party modules and `scenario` is first-party, so there's a gap between them (and ruff will put it back if you take it out).

I guess now that we've put the code in the same repo, there's an argument that both ops and scenario should be considered the same level in this regard, but if we do that it would be better as a separate PR that makes the change everywhere. Since we still release two packages, I'm also not entirely convinced we should do that.

Reply:
Yeah I wasn't sure about this one. I don't think I've really knowingly used subpackages like this before, so I'm not really clear on what the relationship between them should be now.